src/core/libraries/videoout/driver.cpp

```diff
@@ -175,8 +176,11 @@ void VideoOutDriver::Flip(std::chrono::microseconds timeout) {
         requests.pop();
     }
 
-    // Present the frame.
-    renderer->Present(req.frame);
+    if (!renderer->ShowSplash(
+            req.frame)) { // Whatever the game is rendering show splash if it is active
```

Comment:
Nit, the comment can be placed on top of the if statement so clang format doesn't mess up this badly